client/competitionRegistration.js

```diff
@@ -111,7 +111,7 @@ Template.competitionRegistration.helpers({
 
   registrationAskAboutGuests: function() {
     var competitionId = this.competitionId;
-    competition = Competitions.findOne({ _id: competitionId }, { fields: { registrationAskAboutGuests: 1 } });
+    var competition = Competitions.findOne({ _id: competitionId }, { fields: { registrationAskAboutGuests: 1 } });
```

Comment:
RubyMine code inspection found it. I guess we didn't have any other ones. Aside from the Gizmo one I introduced...